schedule/yast/ssh-x@s390x_zvm.yaml

```diff
@@ -0,0 +1,20 @@
+name:           ssh-X
+description:    >
+  Conduct an installation using ssh with X-Forwarding.
+  Might only be effective for zVM and powerVM
+vars:
+  YUI_REST_API: 1
+  DESKTOP: textmode
+  VIDEOMODE: ssh-x
+schedule:
+  access_beta:
+    - installation/access_beta_distribution
+    - installation/ensure_installer_fullscreen
```

Comment:
this is arguable, but the resize seems to be needed from the selection screen, to not mix thing, we could do instead:
```
product_selection:
  - installation/ensure_installer_fullscreen
  - installation/product_selection/install_SLES
```

Reply:
changed.